src/Servers/Kestrel/Core/src/Internal/Http/HttpProtocol.Generated.cs

```diff
@@ -369,137 +371,137 @@ private void ExtraFeatureSet(Type key, object value)
                 }
                 else
                 {
-                    ExtraFeatureSet(key, value);
+                    ExtraFeatureSet(key, value!); // TODO: What happens if you set an extra feature with a null value?
```

Comment:
Do you plan to file an issue to figure this out?

Reply:
I'm going to fix it in this PR